build.gradle

```diff
@@ -22,10 +22,11 @@ repositories {
 
 dependencies {
     api "org.spigotmc:spigot-api:${gradle.ext.apiVersionFull}"
-    implementation 'org.junit.jupiter:junit-jupiter:5.7.1'
-    implementation 'org.hamcrest:hamcrest-library:1.3'
-    implementation 'org.apache.commons:commons-io:1.3.2'
-    implementation 'org.jetbrains:annotations:19.0.0'
+    implementation 'org.junit.jupiter:junit-jupiter:5.7.2'
+    implementation 'org.hamcrest:hamcrest-library:2.2'
+    //noinspection GradlePackageUpdate
```

Comment:
Mind elaborating this comment?

Reply:
I mean it can be deleted :D was just for ignoring the hint that there is a "newer" version (which isn't the newer version) for that dependency. In my opinion, it shouldn't be a problem but that should the maintainers say somewhere if that kinda comment shouldn't be added.

Reply:
I would say "remove the comment" then, we do wanna update these from time to time, so better have our IDEs hinting at it then us completely forgetting about that